core/arch/arm/plat-rpi3/platform_flags.mk

```diff
@@ -0,0 +1,20 @@
+# 32-bit flags
```

Comment:
What is right and what is wrong here. I can only see `platform_flags.mk` in this commit/folder, but still we refer to it in the `ta/ta.mk`. I.e, this is something used to propagate the platform flags, when building the TA's. But, this isn't something used anylonger? We're exporting it using the `conf.mk` instead. right? I.e, both the `platform_flags.mk` in this PR and the line in `ta/ta.mk` should be removed?

```
$ rgrep platform_flags.mk
ta/ta.mk:       $(wildcard core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk) \
```